recipe/meta.yaml

```diff
@@ -35,7 +35,7 @@ about:
   license: LGPL-2.0-or-later
   license_family: LGPL
   license_file: LICENSE.txt
```

Comment:
@isuruf Apple claims this is under Apache license
https://github.com/apple/cups/blob/master/LICENSE

Are you informed on the history of the decision to make this LGPL?